alpaca/broker/models/journals.py

```diff
@@ -58,6 +58,7 @@ class Journal(ModelWithID):
     transmitter_financial_institution: Optional[str] = None
     transmitter_timestamp: Optional[str] = None
     currency: Optional[SupportedCurrencies] = None
+    created_at: Optional[datetime] = None
```

Comment:
Thank you. Could you please add `created_at` json field [1] into the test cases [2]?

*1 something below
```
"created_at": "2024-05-30T09:13:03.642208Z"
```

*2 https://github.com/alpacahq/alpaca-py/blob/cf798a0b2860d83db593d91edcaae282c805de32/tests/broker/broker_client/test_journal_routes.py#L41